src/Bookings.js

```diff
@@ -1,21 +1,32 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+//import FakeBookings from "./data/fakeBookings.json";
+import SearchButton from "./SearchButton.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState(null);
```

Comment:
Not strictly necessary but usually it's better to initialise with the same data type, in this case an array.

If you change it make sure that your `if` statement on line 19 is also adjusted.